modules/profile/manifests/st2server.pp

```diff
@@ -31,6 +31,13 @@
   $_root_cli_uid = 2000
   $_root_cli_gid = 2000
 
+  # Syslog user differs based on distro
+  $syslog_user = $osfamily ? {
+    'Debian'  => 'syslog',
+    'Ubuntu'  => 'syslog',
```

Comment:
make sure to `::osfamily`.

Also, ubuntu falls under the `Debian` OS family, so L37 won't ever be evaluated.